### PR TITLE
Add loki reduction notebook

### PR DESCRIPTION
**Pull request by nvaytet**
Add a notebook that illustrates how to use the `LokiAtLarmorWorkflow` to compute $I(Q)$ for 4 different samples.
We use the result from the direct beam iterations notebook as the direct beam function for the normalization.

### CODE REVIEW

File: docs/user-guide/loki/loki-iofq.ipynb
Comment (YooSunYoung) [inline docs/user-guide/loki/loki-iofq.ipynb:18]:
```suggestion
    "- [ISIS polymer](#Compute-I(Q))\n",
    "- [AgBeh](#AgBeh-sample)\n",
    "- [Porous silica](#Porous-Silica)\n",
    "- [deut-SDS](#Deut-SDS)"
```

Can we have a link to the title for each samples like this...?

**Review by YooSunYoung — APPROVED** [review]

**Comment by SimonHeybrock** [comment]
Please wait until #163 is merged, so that long-standing PR will not experience conflicts.

**Pull request merged**
